web/src/components/organisms/Project/settings/index.tsx

```diff
@@ -0,0 +1,186 @@
+import { ExclamationCircleOutlined } from "@ant-design/icons";
+import styled from "@emotion/styled";
+import Button from "@reearth-cms/components/atoms/Button";
+import Form from "@reearth-cms/components/atoms/Form";
+import Input from "@reearth-cms/components/atoms/Input";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import Typography from "@reearth-cms/components/atoms/Typography";
+import MoleculeHeader from "@reearth-cms/components/molecules/Common/Header";
+import ProjectMenu from "@reearth-cms/components/molecules/Common/projectMenu";
+import WorkspaceCreationModal from "@reearth-cms/components/molecules/Common/WorkspaceCreationModal";
+import TextArea from "antd/lib/input/TextArea";
+import Layout, { Header, Content } from "antd/lib/layout/layout";
+import Sider from "antd/lib/layout/Sider";
+import React, { useCallback, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+import useDashboardHooks from "../../Dashboard/hooks";
+
+import useHooks from "./hooks";
+
+export interface FormValues {
+  name: string;
+  description: string;
+}
+
+const ProjectSettings: React.FC = () => {
+  const { confirm } = Modal;
+  const [form] = Form.useForm();
+  const { projectId, workspaceId } = useParams();
+  const [collapsed, setCollapsed] = useState(false);
+
+  const {
+    user,
+    personalWorkspace,
+    currentWorkspace,
+    workspaces,
+    handleWorkspaceModalClose,
+    handleWorkspaceModalOpen,
+    workspaceModalShown,
+    handleWorkspaceCreate,
+  } = useDashboardHooks(workspaceId);
+
+  const { project, handleProjectUpdate, handleProjectDelete } = useHooks({
+    projectId,
+  });
+
+  useEffect(() => {
+    form.setFieldsValue({
+      name: project?.name,
+      description: project?.description,
+    });
+  }, [form, project]);
+
+  const handleSubmit = useCallback(() => {
+    form
+      .validateFields()
+      .then(async (values) => {
+        handleProjectUpdate({
+          name: values.name,
+          description: values.description,
+        });
+        form.resetFields();
+      })
+      .catch((info) => {
+        console.log("Validate Failed:", info);
+      });
+  }, [form, handleProjectUpdate]);
+
+  const handleProjectDeleteConfirmation = useCallback(() => {
+    confirm({
+      title: "Are you sure you want to delete this project?",
+      icon: <ExclamationCircleOutlined />,
+      onOk() {
+        handleProjectDelete();
+      },
+      onCancel() {},
```

Comment:
Is the {} correct syntaxy for the confirm's params?

Reply:
Yup, it's an object which has properties, and two methods (onOk, onCancel).

Reply:
@KaWaite , I removed `onCancel` as it is not used in my case